commands/board/list.go

```diff
@@ -118,6 +120,8 @@ func identify(pm *packagemanager.PackageManager, port *commands.BoardPort) ([]*r
 		boards = append(boards, &rpc.BoardListItem{
 			Name: board.Name(),
 			FQBN: board.FQBN(),
+			VID:  port.IdentificationPrefs.Get("vid"),
+			PID:  port.IdentificationPrefs.Get("pid"),
```

Comment:
Why just VID & PID, and not the whole `identificationPrefs`?
In this case we have only vid/pid but the pluggable discovery can provide much more information.